<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ICG Paint</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::EditCopy" resource="../../resources.qrc">
    <normaloff>:/assets/assets/no.png</normaloff>:/assets/assets/no.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRender_Hint">
     <property name="title">
      <string>Render Hint</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditPaste">
       <normalon>:/icons/assets/icons/svg/solid/airplay.svg</normalon>
      </iconset>
     </property>
     <addaction name="actionAntialiasing"/>
     <addaction name="actionTextAntialiasing"/>
     <addaction name="actionSmoothPixmapTransform"/>
     <addaction name="actionVerticalSubpixelPositioning"/>
     <addaction name="actionLosslessImageRendering"/>
     <addaction name="actionNonCosmeticBrushPatterns"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="menuRender_Hint"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Filter</string>
    </property>
    <widget class="QMenu" name="menuPixel">
     <property name="title">
      <string>Pixel</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuMatrix">
     <property name="title">
      <string>Matrix</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuBasics">
     <property name="title">
      <string>Basics</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuDithering">
     <property name="title">
      <string>Dithering</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionApply"/>
    <addaction name="separator"/>
    <addaction name="menuBasics"/>
    <addaction name="menuPixel"/>
    <addaction name="menuMatrix"/>
    <addaction name="menuDithering"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionRZ"/>
    <addaction name="actionZoomFit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="actionShowWindowToolbar"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionPreviousImage"/>
    <addaction name="actionNextImage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuZoom"/>
   <addaction name="menuImage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="whatsThis">
    <string>sidebar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::DockWidgetArea::LeftDockWidgetArea|Qt::DockWidgetArea::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Config</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="dockLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QQuickWidget" name="quickWidget">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="resizeMode">
          <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
         </property>
         <property name="source">
          <url>
           <string/>
          </url>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icons/assets/icons/svg/solid/eye.svg</normalon>
              <disabledoff>:/icons/assets/icons/svg/solid/eye-off.svg</disabledoff>
              <disabledon>:/icons/assets/icons/svg/solid/eye-off.svg</disabledon>
             </iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="applyButton">
            <property name="text">
             <string>APPLY</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/folder-open.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open image file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/document-download.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save image file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/folder-download.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save image file with other file name</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/assets/assets/icons/line.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Draw line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/assets/assets/icons/fill.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
   <property name="toolTip">
    <string>Fill some area</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionStamp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/assets/assets/icons/shape.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Stamp</string>
   </property>
   <property name="toolTip">
    <string>Stamp figure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/alert-circle.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About author</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/github-circle.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/plus-circle.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Resize canvas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/minus-circle.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/adjustments-horizontal.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Config Sidebar</string>
   </property>
  </action>
  <action name="actionApply">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/feather.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Apply Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="actionResetZoom">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/dashboard-icons.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Reset zoom</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/next.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/prev.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionShowToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Toolbar</string>
   </property>
  </action>
  <action name="actionRZ">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/maximaize-circle.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom reset</string>
   </property>
  </action>
  <action name="actionNextImage">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/arrow-right-circle.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPreviousImage">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/arrow-left-circle.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionShowWindowToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/adjustments.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Toolbar</string>
   </property>
  </action>
  <action name="actionZoomFit">
   <property name="icon">
    <iconset>
     <normalon>:/icons/assets/icons/svg/solid/minimaize-circle.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom fit</string>
   </property>
  </action>
  <action name="actionAntialiasing">
   <property name="text">
    <string>Antialiasing</string>
   </property>
  </action>
  <action name="actionTextAntialiasing">
   <property name="text">
    <string>TextAntialiasing</string>
   </property>
  </action>
  <action name="actionSmoothPixmapTransform">
   <property name="text">
    <string>SmoothPixmapTransform</string>
   </property>
  </action>
  <action name="actionVerticalSubpixelPositioning">
   <property name="text">
    <string>VerticalSubpixelPositioning</string>
   </property>
  </action>
  <action name="actionLosslessImageRendering">
   <property name="text">
    <string>LosslessImageRendering</string>
   </property>
  </action>
  <action name="actionNonCosmeticBrushPatterns">
   <property name="text">
    <string>NonCosmeticBrushPatterns</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
